tests/periph_i2c/tests/test.py

```diff
@@ -0,0 +1,328 @@
+import periph_i2c_if
+import bpt_if
+import logging
+import argparse
+import time
+
+__BPT_ADDR = 85
+__BPT_USER_REG = 152
+
+
+def kwexpect(val1, val2, level="WARN"):
+    res = level
+    try:
+        if (isinstance(val1, str) and isinstance(val2, str)):
+            if (val1.lower() == val2.lower()):
+                res = "PASS"
+        elif (isinstance(val1, list)):
```

Comment:
you assume val2 is a list, too?

Reply:
Yup but why check, if it isn't it will just get caught in the exception.  Also this is a throwaway test, only as an example.